iroha/Cargo.toml

```diff
@@ -12,7 +12,7 @@ sumeragi-test-mocks = ["iroha_network/mock"]
 [dependencies]
 iroha_derive = { path = "../iroha_macro/iroha_derive" }
 iroha_network = { path = "../iroha_network" }
-async-std = { version = "1.5", features = ["attributes"] }
+async-std = { version = "1.5", features = ["attributes","unstable"] }
```

Comment:
I thought we tried to avoid using unstable?

Reply:
Still waiting for RFC comments: https://wiki.hyperledger.org/display/iroha/Async-std+API+usage
Channels API can be changed later that's why it's under "unstable" feature, but its impl is quite stable https://github.com/async-rs/async-std/commits/master/src/sync/channel.rs and by using it we remove necessity to deal with additional direct dependency on `crossbeam`.

My opinion is that we can give `async-std` a try, but we should make a final decision as a team.